utils/src/ooinstall/cli_installer.py

```diff
@@ -512,13 +512,6 @@ def error_if_missing_info(oo_cfg):
         sys.exit(1)
     oo_cfg.settings['variant_version'] = version.name
 
-    missing_facts = oo_cfg.calc_missing_facts()
-    if len(missing_facts) > 0:
-        missing_info = True
-        click.echo('For unattended installs, facts must be provided for all masters/nodes:')
-        for host in missing_facts:
-            click.echo('Host "%s" missing facts: %s' % (host, ", ".join(missing_facts[host])))
-
```

Comment:
Actually, looking at it closer, a host object can't be created without connect_to set. So I think we're good here.